src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -210,9 +210,9 @@ NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
 route_id | Route/@id | This field is prefixed. 
 line_id | Route/LineRef/@ref | This field is prefixed. 
-route_name | Route/Name | 
+route_name | Route/Name | See [`common.md`](common.md#general-rules) to generate the `name`
```

Comment:
To avoid an ambiguous state with a field indicating we use the `Route/Name` value and the text stating to see the an automatic generation of the name, what would you think of changing the text here to : 
`If Route/Name does not exists or is empty, it is generated automatically. See [...]`

Reply:
Well, in this case, the Netex IDF is not giving any name so we know it is always gonna be empty. I guess what you're looking for is the generic section in [`common.md#general-rules`](https://github.com/CanalTP/transit_model/blob/master/src/documentation/common.md#general-rules) which indeed mention to do the modification only if the name is empty.

Does it answer your concern or did I missunderstood.

Reply:
@prhod , @woshilapin 
Go for merge guys ? It would be nice if it left tomorrow (in release 0.12.1)

Reply:
Sorry for the late answer. This spec declares we read `Route/Name` as the Netex-IDFM element to be read.
@woshilapin I don't understand your point :)

Reply:
And actually this field is read, and with values ​​in the datasets provided.

Reply:
Ok, let's keep it like this :)

Reply:
My bad, I missed that one, I probably mixed things up with the KV1 connector. Good thing you didn't wait on me to merge. Thank you.